public/custom.css

```diff
@@ -851,4 +855,17 @@ body {
 .cc-c-text__danger-icon {
   color: var(--pf-global--danger-color--100);
   vertical-align: -0.125em;
-}
\ No newline at end of file
+}
+
+/* Setting a fixed width similar to the values used in horizontal forms */
+/* var(--pf-c-form--m-horizontal--md__group--GridTemplateColumns)  */
+/* NOTE that 576px is defined as a sass variable in PF source code as min-width: $pf-global--breakpoint--sm */
+@media screen and (min-width: 576px) {
+  .pf-c-content dl.cc-m-column__fixed-width {
+    grid-template-columns: 200px 1fr;
+  }
+}
+
+.pf-c-form.pf-m-horizontal .pf-c-form__group {
```

Comment:
Change this line to:
`.pf-c-form.pf-m-horizontal.cc-m-wide-label .pf-c-form__group {`

By doing this, we are creating a modifier class for the PF form component to create a wider field label.

And then this class `.cc-m-wide-label` would need to be applied to the element with class `pf-c-form`. 

This is a different approach that what I had originally suggested. But it's better than my previous suggestions. The modifier class lets us control which form has a wider label. And placing the modifier class on the form wrapper, rather than each input/label wrapper means we only need to define it once.

Reply:
Thanks. I updated this.